scripts/agave-build-lists.sh

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env bash
+# Defines reusable lists of Agave binary names for use across scripts.
+
+# Source this file to access the arrays
+# Example:
+#   source "scripts/agave-build-lists.sh"
+#   printf '%s\n' "${AGAVE_BINS_DEV[@]}"
+
+
+# Groups with binary names to be built, based on their intended audience
+# Keep names in sync with build/install scripts that consume these lists.
+
+# shellcheck disable=SC2034
+AGAVE_BINS_DEV=(
+  cargo-build-sbf
+  cargo-test-sbf
+  solana-stake-accounts
+  solana-test-validator
+)
+
+AGAVE_BINS_END_USER=(
+  agave-install
```

Comment:
This should probably also include agave-install-init, unless the plan is to deprecate it now

Reply:
Added to the deprecated group if there are no objections. I don't really see the need for it.